geomesa-accumulo/geomesa-accumulo-datastore/src/main/scala/org/locationtech/geomesa/accumulo/index/AttributeIdxStrategy.scala

```diff
@@ -48,20 +48,26 @@ class AttributeIdxStrategy(val filter: QueryFilter) extends Strategy with LazyLo
       throw new IllegalStateException("Attribute index does not support Filter.INCLUDE")
     }
 
+    val disjointDates = (Long.MinValue, Long.MinValue)
```

Comment:
This is fine if this is the constant we want, but for some reason my inclination is to define

```scala
val disjointDates = (Long.MaxValue, Long.MinValue)
```